docs/wpf/region-navigation/basic-region-navigation.md

```diff
@@ -1 +1,43 @@
-# Basic Region Navigation
\ No newline at end of file
+# Basic Region Navigation
+
+Both view injection and view discovery can be considered to be limited forms of navigation. View injection is a form of explicit, programmatic navigation and view discovery is a form of implicit or deferred navigation. However, in Prism, regions have been extended to support a more general notion of navigation, based on URIs and an extensible navigation mechanism.
+
+Navigation within a region means that a new view is to be displayed within that region. The view to be displayed is identified via a URI, which, by default, refers to the name of the view to be created. You can programmatically initiate navigation using the **RequestNavigate** method defined by the **INavigateAsync** interface.
+
+>**Note:** Despite its name, the **INavigateAsync** interface does not represent asynchronous navigation that's carried out on a separate background thread. Instead, the **INavigateAsync** interface represents the ability to perform pseudo-asynchronous navigation. The **RequestNavigate** method may return synchronously following the completion of navigation operation, or it may return while a navigation operation is still pending, as in the case where the user needs to confirm the navigation. By allowing you to specify callbacks and continuations during navigation, Prism provides a mechanism to enable these scenarios without requiring the complexity of navigating on a background thread.
+
+The **INavigateAsync** interface is implemented by the **Region** class, allowing you to initiate navigation within that region.
+
+```cs
+IRegion mainRegion = ...;
+mainRegion.RequestNavigate(new Uri("InboxView", UriKind.Relative));
+```
+
+You can also call the **RequestNavigate** method on the **RegionManager**, which allows you to specify the name of the region to be navigated. This convenient method obtains a reference to the specified region and then calls the **RequestNavigate** method, as shown in the preceding code example.
+
```

Comment:
I would like to see an additional code snippet of using a string to navigate, instead of just the URI object.